app/src/main/java/de/dhbw/corona_world_app/datastructure/TimeFramedCountry.java

```diff
@@ -94,4 +94,34 @@ public String toString() {
                 ", pop_inf_ratio=" + Arrays.toString(pop_inf_ratio) +
                 '}';
     }
+
+    @Override
+    public int compareTo(TimeFramedCountry o) {
+        int infectedAvg = 0;
+        int deathsAvg = 0;
+        int recoveredAvg = 0;
+        if (o.getInfected().length == this.infected.length && o.getDeaths().length == this.deaths.length && o.getRecovered().length == this.recovered.length && o.getDates().length == this.dates.length) {
+            if (o.getInfected().length > 1) {
+                for (int i = 0; i < o.getInfected().length; i++) {
+                    infectedAvg += o.getInfected()[i] - this.infected[i];
+                }
+                infectedAvg = infectedAvg / o.getInfected().length;
+            }
+            if (o.getDeaths().length > 1) {
+                for (int i = 0; i < o.getDeaths().length; i++) {
+                    deathsAvg += o.getDeaths()[i] - this.deaths[i];
+                }
+                deathsAvg = deathsAvg / o.getDeaths().length;
+            }
+            if (o.getRecovered().length > 1) {
+                for (int i = 0; i < o.getRecovered().length; i++) {
+                    recoveredAvg += o.getRecovered()[i] - this.recovered[i];
+                }
+                recoveredAvg = recoveredAvg / o.getRecovered().length;
+            }
+            return infectedAvg * 10 + deathsAvg * 5 + recoveredAvg;
+        } else {
+            throw new IllegalArgumentException("The input TimeFramedCountry is of different length than this object!");
+        }
```

Comment:
i would use this in a custom comparator instead of defining it as the natural order but is also alright